fuzzers/angora/src/extra_abilist.txt

```diff
@@ -0,0 +1,15 @@
+fun:make_fcontext=uninstrumented
```

Comment:
Where does this file come from? Is this an Angora thing?

Reply:
PHP includes Zend now, which has some assembly code in `repo/Zend/asm/`. These codes are not instrumented nor modeled by Angora, thus ignored.

Reply:
Ah ok cool, makes sense! Is it possible to do something to what we already to do handle inline asm in openssl? See https://github.com/HexHive/magma/blob/v1.2/targets/openssl/src/abilist.txt

If this is specific to php, I suggest moving into the target.

Reply:
Sure, let me see if I can get it done tonight. (Somehow Angora can take ~1 hour to compile PHP)